awesomplete.js

```diff
@@ -301,9 +309,7 @@ function $$(expr, con) {
 	return slice.call((con || document).querySelectorAll(expr));
 }
 
-$.create = function(tag, o) {
-	var element = document.createElement(tag);
-
+$.set = function(element, o) {
```

Comment:
I think it's a useful helper, very similar to what I have in other projects. In fact, I plan to clean up & release these helpers as a standalone library very soon :)
That said, if we have a `$.set` we don't really need a `$.create`, we can just use the native `document.createElement` if needed, then call `$.set()` on it. I think that will also simplify our code a bit, since we can just set `this.dropdown` to whatever, then call `$.set()` on it outside the conditional.